h/static/scripts/controllers/search-bar-controller.js

```diff
@@ -175,22 +203,16 @@ class SearchBarController extends Controller {
     };
 
     /**
-     * Add incomplete search query terms which start with a quote but are missing the end quote
-     * to the input field from the hidden input field on page load.
-     */
-    var addIncompleteSearchTermToInput = () => {
-      this._input.value = SearchTextParser.getIncompleteInputValue(this._inputHidden.value);
-    };
-
-    /**
-     * Create lozenges for the search query terms already in
-     * the hidden input field on page load.
+     * Create lozenges for the search query terms already in the input field on
+     * page load.
      */
-    var lozengifyHiddenInput = () => {
-      var queryTerms = SearchTextParser.getLozengeValues(this._inputHidden.value);
-      queryTerms.forEach(function(term) {
-        addLozenge(term);
+    var lozengifyInput = () => {
+      var {lozengeValues, incompleteInputValue} = SearchTextParser.getLozengeValues(this._input.value);
+      lozengeValues.forEach(function(value) {
```

Comment:
Hmm, this is existing code (I just changed the variable name) but I may as well put in this tweak while I'm at it...